internal/server/server.go

```diff
@@ -392,3 +396,30 @@ func configureEmail(options Options) {
 		email.SMTPServer = options.SMTPServer
 	}
 }
+
+func DefaultOptions(infraDir string) Options {
+	return Options{
+		Version:                  0.2, // update this as the config version changes
+		TLSCache:                 filepath.Join(infraDir, "cache"),
+		DBEncryptionKey:          filepath.Join(infraDir, "sqlite3.db.key"),
+		DBEncryptionKeyProvider:  "native",
+		EnableTelemetry:          true,
+		SessionDuration:          24 * time.Hour * 30, // 30 days
+		SessionExtensionDeadline: 24 * time.Hour * 3,  // 3 days
+		EnableSignup:             false,
+		BaseDomain:               "",
+		EnableLogSampling:        true,
+
+		Addr: ListenerOptions{
+			HTTP:    ":80",
+			HTTPS:   ":443",
+			Metrics: ":9090",
+		},
+
+		DB: data.Options{
+			MaxOpenConnections: 1000,
+			MaxIdleConnections: 1000,
```

Comment:
looks good. want to make the defaults something smaller?

Reply:
Oh ya, I think we had talked about defaulting MaxOpen to 100 if that's what Postgres uses as a default. Is that right? MaxIdle could probably be even smaller.